File: algorithm/src/main/java/baekjoon/p05724/Main.java
package baekjoon.p05724;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (true) {
            int input = sc.nextInt();
            if (input == 0) break;
            System.out.println(solve(input));
        }
    }

    private static int solve(int n) {
        int ret = 0;
        for (int i = 1; i <= n; i++) {
            int nn = (n - i + 1);
            ret += (nn * nn);
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p05724/Main.kt
package baekjoon.p05724

fun main(args: Array<String>) {
    while (true) {
        val input = readLine()!!.toInt()
        if (input == 0) break
        println(solve(input))
    }
}

private fun solve(n: Int): Int {
    return (1..n).sumBy { val i = (n - it + 1);i * i }
}